marklogic-data-hub-spark-connector/src/test/resources/custom-ingestion-endpoint/endpoint.api

```diff
@@ -3,13 +3,7 @@
     "desc": "This is for testing a custom ingestion endpoint",
     "params": [
         {
-            "name": "endpointState",
-            "datatype": "jsonDocument",
-            "multiple": false,
-            "nullable": true
-        },
-        {
-            "name": "workUnit",
+            "name": "endpointConstants",
```

Comment:
This custom endpoint should still have endpointState in it; that's a critical part of verifying that if Ernie uses a custom endpoint, he can define both endpointState and endpointConstants in it.